### DISCUSSION TITLE
I want new color

### DISCUSSION THREAD

**Discussion by vishvajitravalji** [twilio-labs/paste#3179] [General] [answered]
I want to add new color according to Flighthub requirement for this color: rgb(16, 173, 222)

**Comment by TheSisb** [comment, marked as answer]
Hi @vishvajitravalji,

The Paste design system is designed to only allow tokens for styling colors, borders, backgrounds, text, and more. You can [learn about design tokens on our website](https://paste.twilio.design/tokens) and see [our existing list of tokens](https://paste.twilio.design/tokens/list) as well.

In order to use a new color, you need to know _how_ you want to use it. Is it a text color, a background color, a border color, or all three? Once you know that, you can leverage the [Paste Customization Provider](https://paste.twilio.design/customization) to override specific tokens to use new values. The Customization Provider essentially allows you to make an entirely new [Theme](https://paste.twilio.design/theme) in Paste. While this may seem like overhead for simple changes, it allows our system to be used and upgraded safely for hundreds of apps, and is a prevailing trend among design systems these days.

We also have a demo app that can help you build out your own custom theme called [Paste Remix](https://remix.twilio.design/), which will hopefully be of some use to you.

Hope this helps!
